modules/rosa-updating-rosa-cli.adoc

```diff
@@ -0,0 +1,60 @@
+:_content-type: PROCEDURE
```

Comment:
This module is missing some metadata at the top where it says which assemblies it's included in. Can you please add that in? (You can take a look at the other modules in this PR to see what it should look like)

Reply:
Corrected, thankyou!